runSingleTest.sh

```diff
@@ -29,4 +29,6 @@ echo "PID: $NODE_PID"
 echo "Running test $file"
 cmd="npx hardhat test $file --network development"
 $cmd
-kill $NODE_PID
\ No newline at end of file
+code=$?
```

Comment:
what's the significance of this?

Reply:
`$?` this gives us exit code of the last executed command.

If the test failed, we also wanted to return the same exit from the shell script.

We are now returning exit code from this shell script so that we can use it in CI.

Reply:
Oh, right